### PR TITLE
fix(certificates): use real certificate data

### PR DESCRIPTION
**Pull request by douglashall**
This PR depends on https://github.com/edx/edx-platform/pull/19884 which adds a certificate list API endpoint on the backend.

### CODE REVIEW

File: src/services/ProfileApiService.js [hunk 1]
```diff
@@ -131,30 +132,39 @@ export function patchPreferences(username, preferences) {
...
+    //       name: 'Micro Masters',
+    //     },
+    //     title: 'Kazoo Pinch Harmonics',
+    //     organization: 'New England Conservatory',
+    //     downloadUrl: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR88selLYRHDQemO8MnY3w3ajlSuKZcoRhVTp3eGvKq2HMRTn8q',
+    //   },
+    // ];
+    // setTimeout(() => {
+    //   resolve(dummyData);
+    // }, 200);
```
Comment (albemarle) [inline src/services/ProfileApiService.js]: Are we ready to delete the fake data?

Comment (davidjoy) [reply src/services/ProfileApiService.js]: Given that the format of it isn't valid anymore, probably.

**Comment by davidjoy** [comment]
Is there a companion PR to this in edx-platform that we should get in too?

**Comment by douglashall** [comment]
@davidjoy Can you give this one another look now that it is ready for review?

**Pull request merged**